servant-util-beam-pg/src/Servant/Util/Beam/Postgres/Sorting.hs

```diff
@@ -45,10 +44,10 @@ instance IsSql92OrderingSyntax syntax =>
 
 -- | Applies 'orderBy_' according to the given sorting specification.
 sortBy_
-    :: ( backend ~ BeamSortingBackend syntax0 s0
+    :: ( backend ~ BeamSortingBackend syntax s
```

Comment:
There was some reason for these to be `syntax0` and `s0`, although I don't remember which one.
If it works for you, let it be.